serving/docker/scripts/install_inferentia2.sh

```diff
@@ -0,0 +1,47 @@
+apt-get update -y
+apt-get install -y wget \
+                   curl \
+                   gnupg2 \
+                   python3-pip \
+                   git
+
+# Configure Linux for Neuron repository updates
+. /etc/os-release
+tee /etc/apt/sources.list.d/neuron.list > /dev/null <<EOF
+deb https://apt.repos.neuron.amazonaws.com ${VERSION_CODENAME} main
+EOF
+wget -qO - https://apt.repos.neuron.amazonaws.com/GPG-PUB-KEY-AMAZON-AWS-NEURON.PUB | apt-key add -
+
+apt-get update -y && apt-get install aws-neuronx-dkms=2.* \
+                                     aws-neuronx-collectives=2.* \
+                                     aws-neuronx-runtime-lib=2.* \
+                                     aws-neuronx-tools=2.* -y
+
+export PATH=/opt/aws/neuron/bin:$PATH
+
+# Install pip
+ln -sf /usr/bin/python3.8 /usr/bin/python
+curl -O https://bootstrap.pypa.io/get-pip.py
+python3 get-pip.py
+python3 -m pip install -U pip
+rm get-pip.py
+
+# Set pip repository pointing to the Neuron repository
+python3 -m pip config set global.extra-index-url https://pip.repos.neuron.amazonaws.com
+
+# Install wget, awscli
+python3 -m pip install wget \
```

Comment:
accelerate and transformers installation should be moving outside, these are not part of the neuronx installation packages.

Reply:
why install wget in python?

Reply:
> accelerate and transformers installation should be moving outside, these are not part of the neuronx installation packages.

Done

> why install wget in python?

Just copied it over from the inferentia docs. Will remove it